src/RockPaperScissors.h

```diff
@@ -1,4 +1,11 @@
 #ifndef ROCKPAPERSCISSORS_H
 #define ROCKPAPERSCISSORS_H
+#include <string>
+using namespace std;
+string RockPaperScissorsPlay(const string player1, const string player2);
```

Comment:
This could use some whitespace
```suggestion

string RockPaperScissorsPlay(const string player1, const string player2);
```